second-edition/src/ch07-02-controlling-visibility-with-pub.md

```diff
@@ -53,27 +53,27 @@ fn main() {
 }
 ```
 
-We use the `extern crate` command to bring the `communicator` library crate
-into scope. Our package now contains *two* crates. Cargo treats *src/main.rs*
-as the root file of a binary crate, which is separate from the existing library
-crate whose root file is *src/lib.rs*. This pattern is quite common for
-executable projects: most functionality is in a library crate, and the binary
-crate uses that library crate. As a result, other programs can also use the
-library crate, and it’s a nice separation of concerns.
+Usamos el comando `extern crate` para llevar el cajón de la biblioteca del `comunicador`
+al alcance. Nuestro paquete ahora contiene *dos* cajones. Cargo trata *src/main.rs*
+como el archivo raíz de un cajón binario, que está separado del cajón de la biblioteca existente
+cuyo archivo raíz es *src/lib.rs*. Este patrón es bastante común para
+proyectos ejecutables: la mayoría de las funciones se encuentran en un cajón de biblioteca, y el cajón
+binario usa esa cajón de biblioteca. Como resultado, otros programas también pueden usar el
+cajón de la biblioteca, y es una buena separación de preocupaciones.
 
-From the point of view of a crate outside the `communicator` library looking
-in, all the modules we’ve been creating are within a module that has the same
-name as the crate, `communicator`. We call the top-level module of a crate the
-*root module*.
+Desde el punto de vista de un cajón fuera de la biblioteca del `comunicator` 
+mirando, todos los módulos que hemos estado creando están dentro de un módulo que tiene el mismo
+nombre que el cajón, `comunicator`. Llamamos al módulo de nivel superior de un cajón el
```

Comment:
Can you please review every instance of 'cajón'? In my opinion would be better to leave the original 'crate' in this case